ArduCopter/Parameters.cpp

```diff
@@ -53,7 +53,7 @@ const AP_Param::Info Copter::var_info[] = {
     // @Param: SYSID_MYGCS
     // @DisplayName: My ground station number
     // @Description: Allows restricting radio overrides to only come from my ground station
-    // @Values: 255:Mission Planner and DroidPlanner, 252: AP Planner 2
+    // @Values: 252: AP Planner 2, 255:Mission Planner and DroidPlanner
```

Comment:
252 is APM Planner 2.

Reply:
I agree, but I think having this field removed is a better option since other vehicles (Rover, Plane etc) don't have it wither.